lib/kubernetes-deploy/kubernetes_resource/customresourcedefinition.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class CustomResourceDefinition < GenericResource
```

Comment:
This class allows `k8s-deploy` to monitor deployment of custom resource *definitions*.